config/php/php-cli.ini

```diff
@@ -0,0 +1,18 @@
+session.auto_start = Off
+short_open_tag = Off
+
+error_reporting = E_ALL & ~E_NOTICE & ~E_WARNING & ~E_STRICT & ~E_DEPRECATED
+
+opcache.enable_cli=1
+opcache.fast_shutdown=1
+opcache.memory_consumption=128
+opcache.interned_strings_buffer=8
+opcache.max_accelerated_files=4000
+opcache.revalidate_freq=60
+# http://symfony.com/doc/current/performance.html
+realpath_cache_size = 4096K
+realpath_cache_ttl = 600
```

Comment:
php.ini can be load in cli context, that way you can avoid duplication.

Reply:
I saw this separation in other projects... in cli context the memory limit is set to `2G`. See line 16 in that file:
```php
memory_limit = 2G
```
So only on cli we have a different setting for `memory_limit`. For the normal usage we did not explicitly set something which then uses the default of `128M`.